packages/fluentui/accessibility-definitions/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@fluentui/accessibility-definitions",
+  "description": "A set of acc.... ",
```

Comment:
This looks unfinished? Also I'm curious why another package ended up being needed.

Reply:
@ecraig12345  thank you for cath. Updated. 

The new package I am adding because of dependency which "definitions" files have inside. 
The "definitions" files depends on "a11y-rules" package. 

"Definitions" files were before together with "behaviors" files in "accessibility" package:
office-ui-fabric-react\packages\fluentui\accessibility\src\behaviors

"Accessibility" package is public.
But we don't want to make "a11y-rules" package public. (as dependency of "definittions" files) 

So this was the reason extract "definitions" files out of the "accessibility" package.

Reply:
I'm still a little confused. If the definitions are only needed as a **devDependency** of `accessibility`, they don't need to be in a published package. But if they're going to be used (directly or indirectly) by consumers of `accessibility`, the packages need to be public. Also, you can't have a private package as (non-dev) dependency of a published package, even indirectly.

Reply:
I think "definitions" we don't want to be consume/used with accessibility package. 

Because this definitions are used only for testing of "Behaviors" files and user of "accessibility" can use only "behaviors" files. 

`Also, you can't have a private package as (non-dev) dependency of a published package, even indirectly.`
if this issue, should I revert changes and put "definitions" package back to the "rules" package and make "rules" package public?
and if it will be public do I need create separate PR for it? :)

Reply:
We discussed this offline, the distinction among `a11y-testing`, `a11y-rules` and `accessibility-definitions` is hard to explain which probably means it does not have much sense. Based on that we agreed to have the two following packages:
- `@fluentui/accessibility` - PUBLIC package, contains behaviors but not their description.
- `@fluentui/a11y-testing` - private package, contains behavior descriptions, testing rules and testing framework for it.